flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -44,7 +44,8 @@ class AndroidNotificationDetails {
     this.visibility,
     this.timeoutAfter,
     this.category,
-    this.additionalFlags,
+    this.isFullScreen = false,
```

Comment:
Rename to more closely match the property on Android side i.e. `fullScreenIntent`. The parameter passed via the method channel should be renamed to match it as well